Assignment6.Rmd

```diff
@@ -95,7 +94,7 @@ table(M2$certified, M2$predict2)
 
 Choose a data file from the (University of Michigan Open Data Set)[https://github.com/bkoester/PLA/tree/master/data]. Choose an outcome variable that you would like to predict. Build two models that predict that outcome from the other variables. The first model should use raw variables, the second should feature select or feature extract variables from the data. Which model is better according to the cross validation metrics?  
 
-The first model that predict outcome (GPA) from all other variables has less cross-validation error than the second model that predict outcome (GPA) from 3 independent variables - `SUBJECT`, `GRD_PTS_PER_UNIT`, `TERM`.
+Since xerror(model1) = 1.14077, xerror(model2) = 0.94064, the second model that predict outcome (GPA) from 3 independent variables - `SUBJECT`, `GRD_PTS_PER_UNIT`, `TERM` has less cross-validation error than the first model that predict outcome (GPA) from all other variables.
```

Comment:
Not sure where get the error rate of 1.14 for model 1. As far as I can see from your html results, the models have a error rate of 0.85 and 0.83 respectively.